cc/tf_utils.h

```diff
@@ -16,6 +16,7 @@
 #define CC_TF_UTILS_H_
 
 #include <string>
+#include <vector>
```

Comment:
was this needed or is this git artifact?

Reply:
It should be here and I included it because I was fixing this files.